src/main/java/com/pusher/client/channel/ChannelEventListener.java

```diff
@@ -40,4 +40,19 @@ public interface ChannelEventListener extends SubscriptionEventListener {
      *            The name of the channel that was successfully subscribed.
      */
     void onSubscriptionSucceeded(String channelName);
+
+    /**
+     * <p>
+     * Callback fired when a subscription success count
+     * of a channel changes
+     * </p>
+     *
+     *
+     * @param channelName
+     *            The name of the channel for which the count changed.
+     * 
+     * @param count
+     *            The latest count of subscribers on the channel.
+     */
+    void onSubscriptionCountChanged(String channelName, int count);
```

Comment:
I can't bring myself to let it go like this. This makes it harder for anyone who wants to upgrade, even if they don't want to use the feature.

I know the way it's currently designed is annoying to work with. But let's create a new interface that extends this. 
e.g. `ChannelWithSubscriptionCountEventListener`

You probably have to add two more interfaces that extend that for private channels too. `PrivateChannelWithSubscriptionCountEventListener`, `PrivateEncryptedChannelWithSubscriptionCountEventListener`

It's pretty ugly but at least it doesn't force everyone to change their implementation.

Sorry for the late feedback.

We will need to rethink and redesign how these event listeners are added in the next major release.